integration/fab3/fab3_test.go

```diff
@@ -52,6 +61,10 @@ var _ = Describe("Fab3", func() {
 		client        *http.Client
 		SimpleStorage helpers.Contract
 	)
+	core := zapcore.NewCore(zapcore.NewConsoleEncoder(zap.NewDevelopmentEncoderConfig()), zapcore.AddSync(GinkgoWriter), zap.DebugLevel)
```

Comment:
if you put this in the fab3_suite_test.go in a before each clause it will apply to all the tests including those of the other files.

Reply:
Does the variable show up in the other contexts or do I have to define a global variable to expose it?

Reply:
you would have to move all four lines, so once you run the ReplaceGlobals it should work on all threads and tests.

Reply:
ya, that's better too cause I can use the zap.S() in any test and have it.